code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -526,6 +538,7 @@
 
 /obj/item/ammo_box/magazine/toy/smg
 	name = "foam force SMG magazine"
+	icon = 'icons/obj/ammo.dmi'
```

Comment:
I just realized that a majority of these don't use the tgmc ammo... please switch back to the original ammo and override with the tgmc icon where needed